modules/meshrenderinggl/include/modules/meshrenderinggl/processors/rasterizationrenderer.h

```diff
@@ -0,0 +1,171 @@
+ * value
+ *       + __Normal Source__ Source of the pixel normal: interpolated or not
+ *       + __Shading Mode__ The shading that is applied to the pixel color
+ *       + __Show Edges__ Show triangle edges
+ *       + __Edge Color__ The color of the edges
+ *       + __Edge Opacity__ Blending of the edge color:
+ *           0-1: blending factor of the edge color into the triangle color, alpha unmodified;
+ *           1-2: full edge color and alpha is increased to fully opaque
+ *   * __Back Face__ Settings for the back face
+ *       + __Show__ Shows or hides that face (culling)
+ *       + __Same as front face__ use the settings from the front face, disables all other settings
+ * for the back face
+ *       + __Copy Front to Back__ Copies all settings from the front face to the back face
+ *       + __Color Source__ The source of the color: vertex color, transfer function, or external
+ * constant color
+ *       + __Separate Uniform Alpha__ Overwrite alpha settings from above with a constant alpha
+ * value
+ *       + __Normal Source__ Source of the pixel normal: interpolated or not
+ *       + __Shading Mode__ The shading that is applied to the pixel color
+ *       + __Show Edges__ Show triangle edges
+ *       + __Edge Color__ The color of the edges
+ *       + __Edge Opacity__ Blending of the edge color:
+ *           0-1: blending factor of the edge color into the triangle color, alpha unmodified;
+ *           1-2: full edge color and alpha is increased to fully opaque
+ */
+
+/**
+ * \class RasterizationRenderer
+ * \brief Mesh Renderer specialized for rendering highly layered and transparent surfaces.
+ *
+ * It uses the FragmentListRenderer for the rendering of the transparent mesh.
+ * Many alpha computation modes, shading modes, color modes can be combined
+ * and even selected individually for the front- and back face.
+ */
+class IVW_MODULE_MESHRENDERINGGL_API RasterizationRenderer : public Processor {
+public:
+    RasterizationRenderer();
+    virtual ~RasterizationRenderer() = default;
+
+    virtual const ProcessorInfo getProcessorInfo() const override;
+    static const ProcessorInfo processorInfo_;
+
+    /**
+     * \brief Performs the rendering.
+     */
+    virtual void process() override;
+
+protected:
+    RasterizationInport rasterizations_;
+
+    std::shared_ptr<ImageInport> imageInport_;
+    ImageOutport outport_;
+
+    BoolProperty forceOpaque_;
+
+    struct IllustrationSettings {
+        IllustrationSettings();
+
+        BoolCompositeProperty enabled_;
+        FloatVec3Property edgeColor_;
+        FloatProperty edgeStrength_;
+        FloatProperty haloStrength_;
+        IntProperty smoothingSteps_;
+        FloatProperty edgeSmoothing_;
+        FloatProperty haloSmoothing_;
+
+        FragmentListRenderer::IllustrationSettings getSettings() const;
+    };
+    IllustrationSettings illustrationSettings_;
+
+    FragmentListRenderer flr_;
+    typename Dispatcher<void()>::Handle flrReload_;

```

Comment:
use using

Reply:
I fell for that btw: It's typename, not typedef...

Reply:
ahh... can't read...